runtime/nls/exel/exelib_ca.nls

```diff
@@ -944,4 +944,16 @@ J9NLS_EXELIB_INTERNAL_HELP_XXNOPORTABLESHAREDCACHE.system_action=
 J9NLS_EXELIB_INTERNAL_HELP_XXNOPORTABLESHAREDCACHE.user_response=
 # END NON-TRANSLATABLE
 
+J9NLS_EXELIB_INTERNAL_HELP_XXENABLESHAREORPAH=\ \ -XX: +ShareOrphans permet compartir classes sense informaci\u00f3 de cam\u00ed d'acc\u00e9s de classe/m\u00f2dul com a orfes
```

Comment:
Why the space after `:` (in `-XX: +ShareOrphans`)? I don't see it in other references to `-XX:...` options in this file. See for example, line 934:
```
J9NLS_EXELIB_INTERNAL_HELP_XXPORTABLESHAREDCACHE=\ \ -XX:+PortableSharedCache ...
```

Reply:
We don't control the automated translations. I could fix them manually but the spaces might come back next time. I think I'll just close this PR and we can try translations again another day and hope for a better result.

Reply:
Why would automation add them some places, but not others?

Reply:
I suspect they are constantly fiddling with translations. Much of the earlier translation was done by humans. Only recent translations, PRs I've opened, are automated. I think we could get different results by waiting a week or two. Perhaps there is a way to report a problem, but atm I don't know how to do that.

Reply:
The other thing to note is that there is a translation cache. I don't believe it re-translates everything when translation is done, it looks for new things. Since I've submitted this translation now, perhaps it will be stuck this way, but we have seen older translated messages get updated on occasion, so I have some hope.

Reply:
I think we should fix the stray, unwanted spaces, and try to report this misbehavior in hopes they won't come back.

Reply:
What's the plan for this? I still think we should remove the extra spaces (added inconsistently) and hope the automation doesn't suggest adding them back.

Reply:
It does add them back, also it mangles command line options in newer messages than are translated in this PR. I need to ask on Slack about how to work around it. I do have a recent version I've updated manually, but without a work around it means re-doing the fixes every time there is a new translation.

Reply:
Someone is looking into the problem with `-XX:` options. They aren't supposed to be modified. Hopefully afterwards they will look at the issue with `--sun-misc-unsafe-memory-access`.

Reply:
I believe processing the options is fixed. However the front end for submitting translations is broken atm, it's being looked at.